app.js

```diff
@@ -0,0 +1,45 @@
+let Books = JSON.parse(localStorage.getItem('Books' || '[]'));
+if (Books === null) {
+  Books = [];
+}
+
+function removeBook(title) {
+  const book = document.getElementById(title);
+  book.remove();
+  Books = Books.filter((bookObj) => bookObj.title !== title);
+  localStorage.setItem('Books', JSON.stringify(Books));
+}
+
+function addBook(bookObj) {
+  const bookList = document.getElementById('book-list');
+  const book = document.createElement('li');
+  book.setAttribute('id', bookObj.title);
+  book.innerHTML = `
+      <h3> ${bookObj.title} </h3> 
+      <p>  ${bookObj.author} </p>
+      `;
+  const removeBtn = document.createElement('button');
+  removeBtn.innerHTML = 'Remove';
+  removeBtn.addEventListener('click', () => removeBook(bookObj.title));
+  book.appendChild(removeBtn);
+  bookList.appendChild(book);
+}
```

Comment:
- [ ] [Optional] It would be better professionally if you named this function `displayBook` instead of `addBook` because that's what this function do.

Reply:
Thanks for your advice, It's Done.